src/components/MeatItemModal.js

```diff
@@ -48,6 +49,12 @@ const MeatItemModal = ({ order, setOrder, isCartButtonsOnModal, setIsMeatItemMod
     const updateOrder = () => {
         const updatedCartOrders = cartOrders.orders.map(cartOrder_ => {
             if (cartOrder_.orderId === order.orderId) {
+                if (order.addOns) {
+                    if (!order.addOns.length) {
```

Comment:
I think we talked about this section and the purpose of the delete, that addons shouldn't be there in the first place.
Also, this is not how you write a double condition, you use `&&`